addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -22,6 +22,13 @@
 				<label>Address</label>
 				<description>The address of the AVR to control.</description>
 			</parameter>
+            <parameter name="PORT" type="integer" required="false">
```

Comment:
XML files should be indented using tabs according to our code formatter, this looks mixed. Please reformat from Eclipse